components/tidb_query/src/rpn_expr/impl_like.rs

```diff
@@ -23,6 +23,25 @@ pub fn like(
     }
 }
 
+#[rpn_fn]
+#[inline]
+pub fn regexp(target: &Option<Bytes>, pattern: &Option<Bytes>) -> Result<Option<i64>> {
```

Comment:
Hi, for RegExp, please also implement it for constant patterns. You can refer the implementation from the documentation in `rpn_function.rs`.

Reply:
Hi, does the `constant patterns` stored in `ctx` ?
Or I just add a `regex_match` function for constant patterns?

Reply:
You should implement `Regex_Fn` trait manually as described [here](https://github.com/tikv/tikv/blob/master/components/tidb_query_codegen/src/rpn_function.rs#L128)